apps/chromium-browser/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  chromium:
+    image: lscr.io/linuxserver/chromium:stable
+    container_name: chromium
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
+      - CHROME_CLI=https://www.linuxserver.io/ #optional
+    volumes:
+      - /path/to/config:/config
```

Persisted data needs to be bound to the app data directory like this:
```suggestion
      - ${APP_DATA_DIR}/data:/config
```
Please also create a directory called `data` in `apps/chromium-browser/data`. Inside of that directory please create a file called `.gitkeep`. The reason is, that docker would create that directory for us automatically. But the permissions would be wrong, so we have to create it beforehand.